README.md

```diff
@@ -22,10 +22,15 @@ Carousel component used to render carousel items via `renderItem` prop.
 
 ```js
 type CarouselProps = {
-  // style for the FlatList content container element
+  // applied to the content container within FlatList
+  // |------------ [ style ]--------------------------|
+  // | |---------- [ contentContainerStyle ] -------| |
```

Comment:
it would actually be style > flatListStyle > contentContainerStyle for these diagrams